hack/get-kustomize.sh

```diff
@@ -5,21 +5,24 @@
 
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 KUSTOMIZE_VERSION="v3.9.1"
+_BINDIR=$THIS_DIR/.bin
 
 # Check if you have kustomize installed, or download it.
 # please note that different versions of kustomize can give different results
 check_or_install_kustomize() {
     # Check if there is already a kustomize binary in $_BINDIR and if yes, check
     # if the version matches the expected one.
```

Comment:
`if yes, check if the version matches the expected one.`

Do we do this?

Reply:
No, but we should because they have an habit of doing backward incompatible changes.

Reply:
PR updated